tests/Unit/Pypp/PyppPyTests.py

```diff
@@ -55,8 +55,7 @@ def ndarray_of_floats():
     import numpy as np
     return np.array([1., 2.], dtype='float32')
 
-# Test conversion from numpy array to tensors
-
+# These are used to check converting to a Tensor works
```

Comment:
Experience from SpEC has shown that it's good to avoid the words "these" and "this" in comments in favor of people specific. I'd prefer "The functions scalar, vector, tnsr* are used..."